sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -863,9 +861,9 @@ struct _pi_buffer final : _pi_mem {
   // Buffer/Sub-buffer constructor
   _pi_buffer(pi_context Ctx, char *Mem, char *HostPtr,
              _pi_mem *Parent = nullptr, size_t Origin = 0, size_t Size = 0,
-             bool MemOnHost = false)
-      : _pi_mem(Ctx, HostPtr, MemOnHost), ZeMem{Mem}, SubBuffer{Parent, Origin,
-                                                                Size} {}
+             bool MemOnHost = false, bool ImportedHostPtr = false)
```

Comment:
I think you'd better just initialize to false in-class since there is no intention to ever pass "true" here

Reply:
I thought of that, but if you look at memBufferCreate, you'll see that when imported, a "true" is passed at buffer creation.

Reply:
I don't see it, can you give a link?